### PR TITLE
adjusted project to what was established with challengeStudio

### PR DESCRIPTION
**Pull request by pananimals**

### CODE REVIEW

File: src/css/styles/_top.scss [hunk 1]
```diff
@@ -56,12 +81,19 @@
   display: inline-block !important;
 }
 
+.l-center-vertical--mobile-menu {
+  position: relative;
+  top: 50%;
+  transform: translateY(-50%);
+}
+
```
Comment (pananimals) [inline src/css/styles/_top.scss]: I reverted styling, to center mobile menu vertically, but IMO it looks better and more natural without this styling. To be discussed.

File: src/css/styles/_top.scss [hunk 2]
```diff
@@ -1,5 +1,10 @@
 .top {
   position: relative;
+  width: 100%;
+  min-height: 500px;
+  overflow: hidden;
+  background: url("../img/banner-bg.png") no-repeat 75%;
```
Comment (michalkowol) [inline src/css/styles/_top.scss]: In future we would need switch too looping video, but it should not be so hard.
Ref: https://slicejack.com/fullscreen-html5-video-background-css/

File: src/css/styles/_top.scss [hunk 3]
```diff
@@ -41,11 +46,34 @@
   padding: 1rem;
 }
 
+.l-center-vertical {
+  position: relative;
+  top: 50%;
+  transform: translateY(-50%);
+}
```
Comment (michalkowol) [inline src/css/styles/_top.scss]: Should we have separate file for layouts. We have one for helpers.

- [ ] separate file for layouts

File: src/css/styles/_top.scss [hunk 4]
```diff
@@ -41,11 +46,34 @@
...
+
+.l-center-vertical--banner {
+  padding-top: 52px;
+}
```
Comment (michalkowol) [inline src/css/styles/_top.scss]: IMHO this is banner specific.

- [ ] move it to banner

Comment (michalkowol) [inline src/css/styles/_top.scss]: Correct me if I am wrong: 52px is height of menu, isn't it?

File: src/index.html [hunk 5]
```diff
@@ -9,59 +9,62 @@
     <meta name="keywords" content= "aurator">
     <meta name="author" content="Softwareberg">
 
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+
     <title>Aurator</title>
 </head>
 <body class="app">
-    <div class="container">
+    <div class="container-fluid">
```
Comment (michalkowol) [inline src/index.html]: When I was learning to use Bootstrap there was requirement to always use `row` and `col`

- [ ] do we need `row` and `col`

File: src/css/main.scss [hunk 6]
```diff
@@ -1,5 +1,35 @@
 @import "styles/bootstrap";
 @import "styles/variables";
 @import "styles/typography";
+@import "styles/helpers";
 @import "styles/general";
 @import "styles/top";
+@import "styles/naprawa";
```
[the remark this answers is not in the document]
Comment (pananimals) [reply src/css/main.scss]: Soon ;)

File: src/css/styles/_general.scss [hunk 7]
```diff
@@ -6,69 +6,16 @@
...
-    font-size: 1rem;
-  }
-
-  .h-txt10 {
-    font-size: 0.6rem;
-  }
+.container {
+  max-width: 1600px;
+  padding: 0 4rem;
+  overflow: hidden;
```
[the remark this answers is not in the document]
Comment (pananimals) [reply src/css/styles/_general.scss]: Moved padding to separate class, removed `overflow: hidden`, `max-width` left 1600px according to the project (but I'm open to change that). Changes already introduced in new branch, now Included in here in latest commit.

**Comment by michalkowol** [comment]
image

File: src/css/styles/_top.scss (hunk 4 above)
Comment (pananimals) [reply src/css/styles/_top.scss]: Unfortunately, scss vars cannot be used inside `calc(*)` expression, on new branch I will get rid of this variable and return to using pixels in its places of use.

**Pull request merged**